web-console/4.0-designs/storage/storage.md

```diff
@@ -0,0 +1,22 @@
+# Storage
+
+## Create Storage Class
+
+![Storage class page with create button](img/create-storage-class.png)
+- Users with appropriate privileges can click to create a new Storage Class.
+
+![create storage class form](img/storage-class-form.png)
+- They are presented with a form that enables them to define the name, description, type of storage, reclaim policy, and provisioner for this storage class.
+
+
+## Attach Storage
+
+![deployment detail page with add storage item](img/add-storage.png)
+-  Deployment pages will contain a new `Add Storage` option within the actions dropdown.
```

Comment:
Should we consider "Add Volume"? Then we can later add an option to the form for config maps or secrets.

FWIW, the CLI uses `oc volume --add`

Reply:
That's a good point. I'll update it